presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.202
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for queries involving aggregations over the result of an outer join
+  (`issue #10592 <https://github.com/prestodb/presto/issues/10592>`_).
+* Fix :func:`map` to fail on duplicate keys, instead of producing a corrupted map silently.
+* Fix OOM in bucketed execution by scheduling new splits on the same worker as
```

Comment:
Let's write this in a more natural way, without relying on "OOM". Something like "Prevent query from running out of memory when ..."

Or is this actually a worker crash?